src/main/java/com/linkedin/kmf/common/Utils.java

```diff
@@ -77,7 +77,7 @@ public static int createMonitoringTopicIfNotExists(String zkUrl, String topic, i
     ZkUtils zkUtils = ZkUtils.apply(zkUrl, ZK_SESSION_TIMEOUT_MS, ZK_CONNECTION_TIMEOUT_MS, JaasUtils.isZkSecurityEnabled());
     try {
       if (AdminUtils.topicExists(zkUtils, topic)) {
-        LOG.info("Monitoring topic \"" + topic + "\" already exists.");
+        LOG.debug("Monitoring topic \"" + topic + "\" already exists.");
```

Comment:
I see. Let's talk to SRE to understand if they want to see exception in this case. They probably don't need it. We can remove it if they do not need it. Otherwise, we can assign these EKG ticket to them. Besides, can you have some example EKG tickets? I would like to see what exception may be thrown there.